pytorch_lightning/accelerators/accelerator.py

```diff
@@ -348,7 +348,12 @@ def connect_precision_plugin(self, plugin: PrecisionPlugin) -> None:
 
     def to_device(self, batch: Any) -> Any:
         """Pushes the batch to the root device"""
-        return self.batch_to_device(batch, self.root_device)
+        # Todo (tchaton) Better fix
+        is_dict = isinstance(batch, dict)
```

Comment:
would it be better to have as not a list?

Reply:
I will look into how to properly solve this bug in another PR. Might need a bit more digging.